chapters/en/module3.md

```diff
@@ -12,19 +12,708 @@ id: 3
+- Set the step size to 100. 
+- Give the area plot an appropriate opacity.
+- Map the `body_mass_g` and the `density` column that was made to the x and y-axis and making sure to specify that `density` is a quantitative variable. 
+- Map the island to the colour (color) channel. 
+- Make sure to give your plot a title.
+
+<codeblock id="03_09a">
+
+- Are you setting `groupby=['island']` and `as_=['body_mass_g', 'density']` in `transform_density()`?
+- Are you setting `steps=100`?
+- Are you setting `opacity=0.5` in `mark_bar()`?
+- Are you specifying `x='body_mass_g'` or `alt.X('body_mass_g')`?
+- Are you specifying `y='density:Q'` or `alt.Y('density:Q')`?
+- Are you specifying `color='island'` or `alt.Color('island')`?
+- Are you setting a title in `properties()`?
+
+</codeblock>
+
+**Question**      
+
+Which island tends to have penguins with a higher body mass?
+
+<choice id="1" >
+<opt text="Biscoe" correct="true">
+
+This island appears to have penguins with a heavier mass. 
+
+</opt>
+
+<opt text="Dream">
+
+This island doesn't have any penguins with a body mass greater than 5000 g. 
+
+</opt>
+
+<opt text="Torgersen" >
+
+This island doesn't have any penguins with a body mass greater than 5000 g.
+
+</opt>
+
+</choice>
+
+
+Now let's facet based on species and see if that gives us some more insight. 
+
+
+Create a density plot for the `body_mass_g` column for penguins located on different islands but this time facet for species 
+
+Tasks: 
+
+- Use the data source `penguins` to make the plot and assign it to an object named `mass_faceted_plot`.
+- Calculate the KDE of the `body_mass_g` column and make sure to split it up by the categorical column `island` and penguin `species`. When you create your new variable for the calculation make sure to name it `density`. 
+- Set the step size to 100. 
+- Give the area plot an appropriate opacity.
+- Map the `body_mass_g` and the `density` column that was made to the x and y-axis and making sure to specify that `density` is a quantitative variable. 
+- Map the island to the colour (color) channel. 
+- Make sure to give your plot a title and set the width of the plot to 150.
+- Facet the plot by the penguin `species`. 
+
+<codeblock id="03_09b">
+
+- Are you setting `groupby=['island','species']` and `as_=['body_mass_g', 'density']` in `transform_density()`?
+- Are you setting `steps=100`?
+- Are you setting `opacity=0.5` in `mark_bar()`?
+- Are you specifying `x='body_mass_g'` or `alt.X('body_mass_g')`?
+- Are you specifying `y='density:Q'` or `alt.Y('density:Q')`?
+- Are you specifying `color='island'` or `alt.Color('island')`?
+- Are you setting a title in `properties()`?
+- Are you faceting with `.facet('species')`?
+
+</codeblock>
+
+
+
+**Question**      
+
+Based on the plot we just made, what did we learn?
```

Comment:
This question would be A LOT easier to answer, if the three histograms in the question above were stacked vertically, instead of being side-by-side horizontally.

Reply:
Also, I can't click on any of these radio buttons.

Reply:
Done and fixed.